packages/platform-sdk-profiles/src/plugins/plugin-registry.models.ts

```diff
@@ -131,6 +135,10 @@ export class RegistryPlugin {
 		return Object.keys(this.#data.versions);
 	}
 
+	public raw() {
```

Comment:
Since this just aliases `getLatestVersion` you could make `getLatestVersion` public and add a proper type hint for it instead of any.